packages/trader/src/Modules/Contract/Components/Sell/sell-info.tsx

```diff
@@ -1,10 +1,14 @@
-import { observer } from 'mobx-react';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Money } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
+import { TContractInfo } from '@deriv/shared';
 
-const SellInfo = ({ contract_info, sell_info }) => (
+type TSellInfo = {
+    contract_info: TContractInfo;
+    sell_info: { transaction_id: number | string; sell_price: number | string };
+};
+
+const SellInfo = ({ contract_info, sell_info }: TSellInfo) => (
```

Comment:
We don't use `SellInfo ` component in project anymore (at least I wasn't able to find it), but I didn't delete it as this situation might change.

Reply:
☢︎